### PR TITLE
Moderation on the show page in admin

### PR DESCRIPTION
**Pull request by h-lame**
For: https://www.pivotaltracker.com/story/show/97012430

Not styled beyond what we do by default so I might ask for some advice from @alexander-bobin on it.  There may be more commits, but that shouldn't stop you from reviewing now.

### CODE REVIEW

**Comment by otlaitil** [comment]
LGTM :+1:

**Comment by mybits** [comment]
LGTM :dart:

File: app/views/admin/moderation/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<%= form_for petition, url: admin_petition_moderation_path(petition), method: :patch do |f| -%>
+  <%= form_row do %>
+    <%= label_tag :moderation_approve, nil do %>
```
Comment (alexander-bobin) [inline app/views/admin/moderation/_form.html.erb]: GDS tend to wrap radio and checkbox groups in a `<fieldset class="inline">` but that can be done later.

**Comment by alexander-bobin** [comment]
The markup looks fine to me. It won't be a big jump to work it into shape when the new styles come into play

**Pull request merged**